src/modules/launcher/Wox.Plugin/Result.cs

```diff
@@ -9,9 +9,16 @@ namespace Wox.Plugin
     public class Result
     {
 
+        private string _title;
         private string _pluginDirectory;
         private string _icoPath;
-        public string Title { get; set; }
+        public string Title { 
+            get { return _title; } 
+            set
+            {
+                _title = value.Replace("\n", " ");

```

Comment:
You will want to remove "\r\n" and "\r" as well.  This could be put into a common helper method for replacing newline charaters.